README.md

```diff
@@ -136,6 +136,37 @@ task default: 'bundle:audit'
 
     $ [sudo] gem install bundler-audit
 
+### Install using bundler
+
+Installation with `gem` (above) is preferred, but if your team 
+requires all gems to be listed in `Gemfile`, it is possible.
+
+```ruby
+# Gemfile
+group :development do
+  gem 'bundler-audit', require: false
+end
+```
+
+### Install using bundler, from source
+
+It is possible, but after installation, you must init the 
+database submodule.
+
+```ruby
+# Gemfile
+group :development do
+  gem 'bundler-audit', github: 'rubysec/bundler-audit', require: false
+end
+```
+
+```bash
+bundle install
+cd `bundle show bundler-audit`
+git submodule update --init
+cd -
+```
```

Comment:
I can't reproduce any problem for these steps. Can you please describe and provide some instructions?

Reply:
Which step do you suspect is unnecessary? The `submodule update`? I recall that fixed an error saying that it couldn't find the "database" (I don't remember the name of the submodule) But, if it's working for you without the `submodule update` then let's drop that part?

Reply:
> Which step do you suspect is unnecessary?

I wrote "for these steps", all steps from this code block. Except the first one, of course.

So, I've deleted all installed `bundler-audit` copies (I hope), ran `bundle install` and `bundle-audit check --update` works.

Reply:
Cool, thanks for testing that. Want me to delete the suspected unnecessary commands?

Reply:
> Cool, thanks for testing that. Want me to delete the suspected unnecessary commands?

I think, yes. I see no need for them. If somebody experiences problems or will to — we can investigate and solve their problems, probably with such lines for installation, but maybe not (in another way, depending on a problem).